BaseTools/Plugin/OverrideValidation/OverrideValidation.py

```diff
@@ -631,6 +703,15 @@ def path_parse():
         if not os.path.normcase(Paths.RegenPath).startswith(os.path.normcase(Paths.WorkSpace.rstrip(os.sep)) + os.sep):
             raise RuntimeError("Module is not within specified Workspace.")
 
+    if Paths.DeprecatedPath is not None:
+        if not os.path.isfile(Paths.DeprecatedPath):
+            raise RuntimeError("Deprecation path module is invalid")
+        # Needs to strip os.sep is to take care of the root path case
+        # For a folder, this will do nothing on a formatted abspath
+        # For a drive root, this will rip off the os.sep
+        if not os.path.normcase(Paths.DeprecatedPath).startswith(os.path.normcase(Paths.WorkSpace.rstrip(os.sep)) + os.sep):
+            raise RuntimeError("Module is not within specified Workspace.")

```

Comment:
Same here - printing the missing module, or the file the missing module is pulled from, would be useful. Any additional information to help the user root cause and resolve will be good.